fineract-provider/src/main/java/org/apache/fineract/portfolio/note/domain/Note.java

```diff
@@ -56,6 +57,7 @@ public class Note extends AbstractAuditableWithUTCDateTimeCustom<Long> {
     @JoinColumn(name = "loan_transaction_id", nullable = true)
     private LoanTransaction loanTransaction;
 
+    @Setter
```

Comment:
You need also @Getter to comply with the JPA specification.

Reply:
Noted.

Reply:
Noted.